### ISSUE TITLE
[Question]: Does model.Fit return any history information in .NET?

### ISSUE THREAD

**Issue by GadgetNutt** [SciSharp/TensorFlow.NET#1048]
### Description

Does model.Fit return any history information in .NET?  Specifically I'm trying to obtain history in VB.NET but I don't see any information pertenent to the training in the history object when I look at it in the Watch.

**Comment by AsakusaRinne** [comment]
The model.fit would include losses and metrics in the history. For example you can use `history["losses"]` yo get the losses.

**Comment by GadgetNutt** [comment]
In VB.NET this doesn't work.   Here is the message at runtime:
System.Collections.Generic.KeyNotFoundException
  HResult=0x80131577
  Message=The given key 'losses' was not present in the dictionary.
  Source=System.Private.CoreLib
When I use these lines:

        Dim hist As History = Me.Model.fit(x_train, y_train, batch_size:=batchSize, epochs:=epochs, verbose:=1, validation_split:=validationSplit)

        Dim trainLoss As List(Of Single) = hist.history("losses").Select(Function(x) CSng(x)).ToList()

When I look at the hist object in Watch it doesn't show anything useful.
